kotlin-reference-server/src/main/kotlin/com/example/sep24/Sep24Util.kt

```diff
@@ -0,0 +1,116 @@
+package com.example.sep24
+
+import com.example.data.PatchTransactionRecord
+import com.example.data.PatchTransactionsRequest
+import com.example.data.Transaction
+import io.ktor.client.*
+import io.ktor.client.call.*
+import io.ktor.client.plugins.contentnegotiation.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.http.*
+import io.ktor.serialization.kotlinx.json.*
+import java.math.BigDecimal
+import kotlin.time.Duration.Companion.seconds
+import kotlinx.coroutines.delay
+import kotlinx.serialization.json.Json
+import mu.KotlinLogging
+import org.stellar.sdk.*
+import org.stellar.sdk.responses.TransactionResponse
+
+object Sep24Util {
+  private val log = KotlinLogging.logger {}
+
+  // TODO: make parameters in this class configurable
+  private val myKey =
+    System.getenv("STELLAR_KEY") ?: "SDYGC4TW5HHR5JA6CB2XLTTBF2DZRH2KDPBDPV3D5TXM6GF7FBPRZF3I"
```

Comment:
lets use environment variables

Reply:
Let make sure to use a different secret key now since this one has been checked into git.

Reply:
I'm using this key as a reference, it's not a problem if it's misused by malicious party (i.e. drained).
For integration tests etc. I would set a different key stored in env

Reply:
Is this the sep-10 signing key? If yes, I suggest that we use the same environment variable name: `SECRET_SEP10_SIGNING_SEED`